app/views/forms/make_live/new.html.erb

```diff
@@ -0,0 +1,52 @@
+<% set_page_title(title_with_error_prefix(t('page_titles.make_live_form'), @make_live_form.errors.any?)) %>
+<% content_for :back_link, govuk_back_link_to(form_path, "Back to create a form") %>
+
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= form_with(model: @make_live_form, url: make_live_create_path) do |f| %>
+      <% if @make_live_form&.errors&.any? %>
+        <%= f.govuk_error_summary %>
+      <% end %>
+
+
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= @make_live_form.form.name %></span>
+        Make your form live
+      </h1>
+
+      <p class="govuk-body">
+        When you make your form live you will get a URL for the form. The form
+        will not be indexed by search engines, so people will not be able to
+        find it easily. Contact your GOV.UK publishing team to publish a link to
+        your form on GOV.UK so people can find it.
+      </p>
+      <p></p>
```

Comment:
Think we should delete this empty paragraph

Reply:
Good spot